lib/js-yaml/loader.js

```diff
@@ -1163,6 +1166,19 @@ function readTagProperty(state) {
     throwError(state, 'tag name cannot contain such characters: ' + tagName);
   }
 
+  if (atLineEnd) {
+    newPosition = state.position;
+
+    while (is_WHITE_SPACE(ch)) {
+      ch = state.input.charCodeAt(++newPosition);
+    }
+
+    if (ch !== 0 && ch !== 0x0A /* LF */ && ch !== 0x0D /* CR */ && ch !== 0x023 /* # */) {
+      state.position = oldPosition;
+      return false;
+    }
+  }
+
```

Comment:
OK, but what if we have something like that:
``` yaml
root:
  !!map &1
  !!str key: value
```
where `!!map &1` are properties of the root mapping.